nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Configuring cluster memory to meet container memory and risk requirements
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Estimating the number of pods your {product-title} nodes can hold
+
+[source,terminal]
+----
+$ oc create sa cluster-capacity-sa
+----
+
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user cluster-capacity-role \
+    system:serviceaccount:default:cluster-capacity-sa
+----
+
+
+[source,terminal]
+----
+$ oc create -f cluster-capacity-job.yaml
+----
+
+## Allowing containers to consume API objects
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f volume-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f volume-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f configmap.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+## Using Init Containers to perform tasks before a pod is deployed
+
+[source,terminal]
+----
+$ oc create -f myapp.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mydb.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f myservice.yaml
+----
+
+## Using port forwarding to access applications in a container
+
+[source,terminal]
+----
+$ oc port-forward <pod> [<local_port>:]<remote_port> [...[<local_port_n>:]<remote_port_n>]
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> [<local_port>:]<remote_port> [...[<local_port_n>:]<remote_port_n>]
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 5000 6000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 8888:5000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> :5000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 0:5000
+----
+
+## Mapping volumes using projected volumes
+
+[source,terminal]
+----
+$ oc create -f <secrets-filename>
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <your_yaml_file>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret-pod.yaml
+----
+
+## Using sysctls in containers
+
+[source,terminal]
+----
+$ oc apply -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit machineconfigpool worker
+----
+
+
+[source,terminal]
+----
+$ oc apply -f set-sysctl-worker.yaml
+----
+
+## Using volumes to persist container data
+
+[source,terminal]
+----
+$ oc set volume <object_selection> <operation> <mandatory_parameters> <options>
+----
+
+
+[source,terminal]
+----
+$ oc set volume <object_type>/<name> [options]
+----
+
+
+[source,terminal]
+----
+$ oc set volume pod/p1
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade Why does this command require a declarative? The command only generates output that is specific to that pod only.

```
 $ oc set volume pod/rails-postgresql-example-1-build
 rails-postgresql-example-1-build
  host path /var/lib/kubelet/config.json as node-pullsecrets
    mounted at /var/lib/kubelet/config.json
  host path /var/lib/containers/cache as buildcachedir
    mounted at /var/lib/containers/cache
  empty directory as buildworkdir
    mounted at /tmp/build
  secret/builder-dockercfg-sgv9p as builder-dockercfg-sgv9p-push
    mounted at /var/run/secrets/openshift.io/push
```

Reply:
@mburke5678 we don't show the declarative alternative for the `oc set volume <object_type>/<name>` command that describes this. I thought it might make more sense to add it for the examples? Or perhaps better to add it for the original command and leave the examples be as they are?